pkg/ccl/spanconfigccl/spanconfigcomparedccl/testdata/basic

```diff
@@ -57,9 +57,9 @@ configs version=legacy offset=41
 ...
 /Table/46                                  database system (host)
 /Table/47                                  database system (host)
-/Table/50                                  range system
-/Table/106                                 num_replicas=7 num_voters=5
-/Table/107                                 num_replicas=7
+/Table/50                                  database system (host)
+/Table/106                                 ignore_strict_gc=true num_replicas=7 num_voters=5 rangefeed_enabled=true
```

Comment:
Looking only at these tests, what this diff indicates is that according to the system config span, the new two tables have configs with `ignore_strict_gc=true rangefeed_enabled=true`. Is that expected? Shouldn't be, right? Since they're not system tables.